app/controllers/orders_controller.rb

```diff
@@ -20,7 +24,7 @@ def create
   private
 
   def orders_params
-    params.require(:order).permit(:employee_id, :reward_id)
+    params.require(:order).permit(:employee_id, :reward_id, :status)
```

Comment:
this shouldn't you probably don't want a user to modify an order's status with a malformed form, right? :)